aiozipkin/aiohttp_helpers.py

```diff
@@ -36,6 +36,15 @@ def middleware_maker(tracer_key=APP_AIOZIPKIN_KEY,
                 span.tag(HTTP_PATH, request.path)
                 span.tag(HTTP_METHOD, request.method.upper())
 
+                peername = request.remote
+                if peername is not None:
+                    if isinstance(ipaddress.ip_address(peername),
```

Comment:
`ipaddress.ip_address(peername)` could raise either `AddressValueError` or `ValueError`.
https://github.com/phihag/ipaddress/blob/master/ipaddress.py#L161

Reply:
`AddressValueError` is derived from `ValueError`

Reply:
Call `ip_address` once.

What to do with UNIX sockets? Should aiozipkin log them?

Reply:
I'd rather do something like that: 
```python
                peername = request.remote
                if peername is not None:
                    kwargs = {}
                    if isinstance(peername, ipaddress.IPv4Address):
                        kwargs['ipv4'] = str(peername)
                    elif isinstance(peername, ipaddress.IPv6Address):
                        kwargs['ipv6'] = str(peername)
                    else:
                        try:
                            peer_ip = ipaddress.ip_address(peername)
                        except (ipaddress.AddressValueError, ValueError):
                            pass  # if we're okay with swallowing the Exception
                        if isinstance(peer_ip, ipaddress.IPv4Address):
                            kwargs['ipv4'] = str(peer_ip)
                        elif isinstance(peer_ip, ipaddress.IPv6Address):
                            kwargs['ipv6'] = str(peer_ip)
                    if kwargs:
                        span.remote_endpoint(None, **kwargs)
```
Done under the assumption `remote_endpoint` expects `str`.
It has the benefit of using `ipaddress`'s formatting/checker.

A better version would be to always transform into a `IPv4Address`/`IPv6Address` and handle them directly. Strings are not good anyway.

But I don't know enough of this project anyway so I may be wrong about that last part.

Reply:
Under the guarantee that `request.remote` is never a `IPv4Address`/`IPv6Address`, it can be simplified as:

```python
                peername = request.remote
                if peername is not None:
                    kwargs = {}
                    try:
                        peer_ip = ipaddress.ip_address(peername)
                    except (ipaddress.AddressValueError, ValueError):
                        pass  # if we're okay with swallowing the Exception
                    if isinstance(peer_ip, ipaddress.IPv4Address):
                        kwargs['ipv4'] = str(peer_ip)
                    elif isinstance(peer_ip, ipaddress.IPv6Address):
                        kwargs['ipv6'] = str(peer_ip)
                    if kwargs:
                        span.remote_endpoint(None, **kwargs)
```